src/Core/src/Platform/Windows/TransformationExtensions.cs

```diff
@@ -18,17 +18,20 @@ public static void UpdateTransformation(this FrameworkElement frameworkElement,
 			double scaleX = view.Scale * view.ScaleX;
 			double scaleY = view.Scale * view.ScaleY;
 
-			frameworkElement.RenderTransformOrigin = new global::Windows.Foundation.Point(anchorX, anchorY);
-			frameworkElement.RenderTransform = new ScaleTransform { ScaleX = scaleX, ScaleY = scaleY };
-
 			if (rotationX % 360 == 0 && rotationY % 360 == 0 && rotation % 360 == 0 &&
 				translationX == 0 && translationY == 0 && scaleX == 1 && scaleY == 1)
```

Comment:
Is there a way this could do slightly better if someone set `ScaleX`/`ScaleY`, but not rotation or translation?